nuclear/module/NUClearModule.cmake

```diff
@@ -23,7 +23,7 @@ function(NUCLEAR_MODULE)
   # Parse our input arguments
   set(options, "")
   set(oneValueArgs "LANGUAGE")
-  set(multiValueArgs "LIBRARIES" "SOURCES" "DATA_FILES")
+  set(multiValueArgs "LIBRARIES" "SOURCES" "DATA_FILES" "TEST_MODULE_DEPENDENCIES")
```

Comment:
I would argue that changes to this file belong in the other PR as test reactors wont work without this and its the other PR that sets up the functionality to allow test reactors to be a thing

Reply:
Excellent point. I have moved these changes up.